src/syscall/exec_linux.go

```diff
@@ -219,11 +219,11 @@ func forkAndExecInChild1(argv0 *byte, argv, envv []*byte, chroot, dir *byte, att
 	locked = true
 	switch {
 	case hasRawVforkSyscall && (sys.Cloneflags&CLONE_NEWUSER == 0 && sys.Unshareflags&CLONE_NEWUSER == 0):
-		r1, err1 = rawVforkSyscall(SYS_CLONE, uintptr(SIGCHLD|CLONE_VFORK|CLONE_VM)|sys.Cloneflags)
+		r1, err1 = rawVforkSyscall(SYS_VFORK, uintptr(SIGCHLD|CLONE_VFORK|CLONE_VM)|sys.Cloneflags)
```

Comment:
I think some documentation should be added here to explain that we use `SYS_VFORK` instead of `SYS_CLONE`, keeping the clone flags so as not to change the rawVforkSyscall function declaration.

Reply:
Done